ccd-definition/AuthorisationCaseField/CareSupervision/solicitor.json

```diff
@@ -33,6 +33,19 @@
     ],
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseFieldID": "courtBundleListV2",
+    "UserRoles": ["[SOLICITOR]", "[SOLICITORA]", "[SOLICITORB]", "[SOLICITORC]", "[SOLICITORD]",
```

Comment:
double check they only need to see court bundles

Reply:
added